src/Awaitable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Interop\Async;
+
+/**
+ * Simple interface for awaitable objects representing the future value of asynchronous operations.
```

Comment:
I'd suggest: `Awaitable object representing the future value of an asynchronous operation.`